packages/base64/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@endo/base64",
+  "version": "0.1.0",
+  "description": "Transcodes base64",
+  "parsers": {
+    "js": "mjs"
+  },
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "type": "module",
+  "main": "./dist/base64.cjs",
+  "module": "./src/main.js",
+  "browser": "./dist/base64.umd.js",
+  "unpkg": "./dist/base64.umd.js",
+  "types": "./types/main.d.ts",
+  "exports": {
+    "import": "./src/main.js",
+    "require": "./dist/base64.cjs",
+    "browser": "./dist/base64.umd.js"
+  },
+  "scripts": {
+    "build": "yarn build:types && yarn build:dist",
+    "build:dist": "rollup --config rollup.config.js",
+    "build:types": "tsc src/*.js --declaration --allowJs --emitDeclarationOnly --outDir types",
+    "clean": "rm -rf dist",
+    "lint": "yarn lint-check",
+    "lint-check": "eslint '**/*.js'",
+    "lint-fix": "eslint --fix '**/*.js'",
+    "prepublish": "yarn clean && yarn build",
+    "test": "ava",
```

Comment:
With regard to test vs qt, I believe that’s actually a special case. Most packages don’t need to build before test, but SES does because some of its tests depend on the build products (the packaging tests).

Reply:
ok. Resolving.